### DISCUSSION TITLE
Enemy submarine Sighting sonar marker for Combat missions (PVP)

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#6276] [Ideas and suggestions]
With pirate missions in the game having approximate location of Pirates, I think this would be a good idea:

Make the opposite team's sub having a vague sighting marker be visible in the same style as pirate missions

The near marker that goes away once the controlled sub is close to the area.

Related #4844

**Comment by Regalis11** [comment]
This has now (finally) been implemented in our private repo, coming up in a future update. :)
